specs/engage-digital_openapi3.yaml

```diff
@@ -7978,6 +8029,109 @@ components:
           items:
             type: string
           type: array
+    Survey:
+      properties:
+        id:
+          type: string
+        created_at:
+          format: date-time
+          type: string
+        updated_at:
+          format: date-time
+          type: string
+        name:
+          description: |-
+            Name displayed in ED admin interface.
+            Is not displayed to end users.
+          type: string
+        link:
+          description: |-
+            Link to the survey on the provider website.
+          type: string
+        active:
+          type: boolean
+        from_name:
+          description: |-
+            Name displayed as sender when survey is sent to end user.
+            Currently only applies to surveys sent by email.
+          type: string
+        instant_send_threshold:
+          description: |-
+            All intervention shorter than this value (in seconds) will have the survey sent instantly.

```

Comment:
Actually it does make sense to me 🤷 

`intervention shorter than this value (in seconds)`
The sentence "this intervention was 5 seconds too short" seems ok to me

Nevertheless, I still changed the wording because it's unclear